qucs/qucs/components/taperedline.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+* taperedline.cpp - Exponentially tapered line implementation
```

Comment:
here and in the `.h`, "Exponentially" should be removed, it's much more general now :grin: